__tests__/server.test.js

```diff
@@ -3,70 +3,69 @@
 const supertest = require('supertest');
 const { server } = require('../src/server');
 const mockRequest = supertest(server);
-const { dbInstance } = require('../src/models/index');
-const { DESCRIBE } = require('sequelize/types/query-types');
-const { INET } = require('sequelize');
+const { dbConnection } = require('../src/models/index');
```

Comment:
Look at your models/index.js file, specifically the database instance you created on line 11 and exported on line 17. What is it named?